torchmetrics/functional/pairwise/manhattan.py

```diff
@@ -36,11 +36,11 @@ def _pairwise_manhatten_distance_update(
     return distance
 
 
-def pairwise_manhatten_distance(
+def pairwise_manhattan_distance(
```

Comment:
I see it was a typo originally, but still, we need to maintain back-compatibility, so can you add a warning and so on...